LICENSE

```diff
@@ -0,0 +1,12 @@
+Source code in this repository is covered by one of two licenses: (i) the GNU General Public License
+(GPL) v3.0 (ii) the Creative Commons BY-NC-ND v4.0. Creative Commons licensed documents are found in
+the docs directory. Other files are GPL licensed.
+
+GPL v3.0: https://github.com/bitwarden/contributing-docs/blob/master/LICENSE_GPL.txt
+
+CC BY-NC-ND v4.0: https://github.com/bitwarden/contributing-docs/blob/master/LICENSE_CC.txt
```

Comment:
Looks good. Only think I notice now is that these have a .txt extension. I dont see that on the other license files? They seem to be extensionless.

Reply:
They are. In my experience, extensionless seems more common, I believe the `.txt` was appended since you develop on Windows. I'm fine adding it for consistency